com.aobuchow.themes.spectrum/css/default_overrides.css

```diff
@@ -41,11 +41,11 @@ Composite#org-eclipse-ui-navigator-ProjectExplorer * {
 
 .MPartStack.active Table {
   background-color: '#com-aobuchow-themes-spectrum-BASE_COLOR';
-  color: #ccc;
```

Comment:
Some expressiveness was lost here. Maybe in the future, a '#com-aobuchow-themes-spectrum-SECONDARY_FONT_COLOR' should be created, which is a slightly muted/greyed out version of the primary font color.